### DISCUSSION TITLE
p/feishin-for-cfpages/

### DISCUSSION THREAD

**Discussion by giscus** [GenshinMinecraft/GMBlog#24] [Announcements]
# p/feishin-for-cfpages/

最近有关 Cloudflare 的是不是有点多

https://blog.highp.ing/p/feishin-for-cfpages/

**Comment by MinatoShiraha** [comment]
总觉得现在有了很多优秀的流媒体平台之后，像这种自建流媒体的服务变的越来越鸡肋，，

**Reply by GenshinMinecraft** [reply]
> 总觉得现在有了很多优秀的流媒体平台之后，像这种自建流媒体的服务变的越来越鸡肋，，

那倒不一定 而且大部分音源也是扒取流媒体的
